services/influxdb/config.go

```diff
@@ -63,6 +63,23 @@ func NewConfig() Config {
 	}
 }
 
+func (c *Config) SetDefaultValues() {
+	if len(c.ExcludedSubscriptions) == 0 {
+		c.ExcludedSubscriptions = map[string][]string{
+			stats.DefaultDatabse: []string{stats.DefaultRetentionPolicy},
+		}
+	}
```

Comment:
I think we should remove the lines. While it would be nice to be able to set this default, I think its worse to not be able to set an empty set.